pkg/util/kms/kms_kmip.go

```diff
@@ -0,0 +1,113 @@
+package kms
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// KMIP client config options
+const (
+	KMIPEndpoint      = "KMIP_ENDPOINT"
+	KMIPSecret        = "KMIP_CERTS_SECRET"
+	KMIPUniqueID      = "UniqueIdentifier"
+	KMIPTLSServerName = "TLS_SERVER_NAME"
+	KMIPReadTimeOut   = "READ_TIMEOUT"
+	KMIPWriteTimeOut  = "WRITE_TIMEOUT"
+	KMPSecret         = "secret"
+	KMIPCACERT        = "ca.crt"
+	KMIPCLIENTCERT    = "client.crt"
+	KMIPCLIENTKEY     = "client.key"
```

Comment:
```suggestion
	KMIPCACERT        = "CA_CERT"
	KMIPCLIENTCERT    = "CLIENT_CERT"
	KMIPCLIENTKEY     = "CLIENT_CERT"
```

can we use upper case format to keep it similar to other configs please? 
and similar to ibm kms 
https://github.com/noobaa/noobaa-operator/pull/834

Cephcsi 
https://github.com/ceph/ceph-csi/pull/3306/files#diff-7b82a5879713c516aac84d8864a56bb41d63e6bfb396c3829cd99f43c7d4ad47

and rook (upcoming) will have uppercase formatting too.